govcd/media_test.go

```diff
@@ -166,3 +166,25 @@ func (vcd *TestVCD) Test_FindMediaAsCatalogItem(check *C) {
 	check.Assert(catalogItem.CatalogItem.Name, Equals, itemName)
 
 }
+
+// Tests System function Refresh
+func (vcd *TestVCD) Test_RefreshMediaImage(check *C) {
+	skipWhenMediaPathMissing(vcd, check)
+	itemName := TestUploadMedia + "6"
```

Comment:
`TestUploadMedia + "6"` is already used in line 143 above. Wouldn't this duplicate during testing?

Reply:
Fxied